compiler/passes/scopeResolve.cpp

```diff
@@ -1877,6 +1877,19 @@ static Symbol* inType(BaseAST* scope, const char* name) {
   return NULL;
 }
 
+// Returns true if the symbol is already present in the vector, false otherwise
+static bool isRepeat(std::vector<Symbol* >& symbols, Symbol* toAdd) {
+  if (symbols.size() > 0) {
```

Comment:
This size check shouldn't be needed.  If the vector is empty the loop will just have 0 iterations.